packages/wallets/src/evm/connectors/smart-wallet/index.ts

```diff
@@ -216,6 +237,22 @@ export class SmartWalletConnector extends Connector<SmartWalletConnectionArgs> {
     return await this.accountApi.isAcountDeployed();
   }
 
+  /**
+   * Get the underlying account smart contract of the smart wallet.
+   * @returns the account smart contract of the smart wallet.
+   */
+  async getAccountContract(): Promise<Account<IAccountCore>> {
+    const signer = await this.getSigner();
+    const network = (await (await this.getProvider()).getNetwork()).chainId;
+    const sdk = ThirdwebSDK.fromSigner(signer, network);
+
+    if (await this.isDeployed()) {
+      throw new Error("Account contract not deployed");
+    }
+    const contract = await sdk.getContract(await signer.getAddress());
+    return contract.account;
```

Comment:
I'm actually tempted to return the whole contract here rather than contract.account. this will be super useful to call custom functions on accounts